src/v2/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx

```diff
@@ -174,19 +187,39 @@ export const ArtworkFilterContextProvider: React.FC<
     initialFilterState
   )
 
+  const [stagedArtworkFilterState, stage] = useReducer(artworkFilterReducer, {})
```

Comment:
In line 187 above there's `initialFilterState`, would it be good to pass that in here as well so that things are in sync? Then we'd be able to pre-populate state based on the URL (if that's desirable).

Reply:
I think this is moot because when the mobile sheet gets opened [it will instantly copy over the current filter state](https://github.com/artsy/force/pull/5658/files#diff-ef0c649e8149fd6269b58e27016cfb2bR46) (whether derived from url string, user actions, or both) so therefore will be in sync by the time the user interacts with it.